libs/CovidDatasets.py

```diff
@@ -67,14 +70,39 @@
     'Wyoming': 'WY'
 }
 
-class Dataset:
-    """Base class of the Dataset objects. Standardizes the output so that data from multiple differet sources can 
-    be fed into the model with as little hassle as possible."""
 
-    _POPULATION_URL = r'https://raw.githubusercontent.com/covid-projections/covid-data-public/master/data/misc/populations.csv'
-    _BEDS_URL = r'https://raw.githubusercontent.com/covid-projections/covid-data-public/master/data/beds-kff/beds.csv'
+local_public_data = tempfile.TemporaryDirectory()
+local_public_data_populated = False
+# public_data_base_url = "https://raw.githubusercontent.com/covid-projections/covid-data-public/master"
+public_data_base_url = 'file://localhost' + local_public_data.name + '/covid-data-public-master'
+
+
+def create_local_copy_public_data():
+    """
+    Creates a local copy of the public data repository. This is done to avoid
+    downloading the file again for each intervention type.
+    """
+    github_zip_url = "https://github.com/covid-projections/covid-data-public/archive/master.zip"
+    logging.info("Creating a Local Copy of {}".format(github_zip_url))
```

Comment:
Can use f-strings here~ `"fCreating a Local copy of {github_zip_url}"`

Reply:
Oh, can I? I am game, I just was concerned perhaps someone was using an older interpreter.

Reply:
Yeah! I think we're all down for 3.7+